x/delegation/keeper/genesis.go

```diff
@@ -50,6 +50,22 @@ func (k Keeper) InitGenesis(
 			}
 		}
 	}
+	for _, association := range gs.Associations {
+		stakerID := association.StakerID
+		operatorAddress := association.Operator
+		// #nosec G703 // already validated
+		stakerAddress, lzID, _ := assetstype.ParseID(stakerID)
```

Comment:
lzID is not clear for reader, let's add todo to replace this with clientChainID.

Reply:
It will be renamed to `clientChainID`